frontend/src/App.tsx

```diff
@@ -102,6 +103,15 @@ const App = (): React.ReactElement => {
                   path={Routes.HOOKS_PAGE}
                   component={HooksDemo}
                 />
+                {/* TODO: change this so it's not hardcoded */}
+                <PrivateRoute
+                  exact
+                  path={Routes.CAMP_OVERVIEW_PAGE.replace(
```

Comment:
Why do we need to replace the ID parameter here? I tried removing the .replace() function and then reloading the page with the given url and it still worked. Removing this here would save us the refactor in the future maybe?

code:
`<PrivateRoute
                  exact
                  path={Routes.CAMP_OVERVIEW_PAGE}
                  component={CampOverview}
                />`

URL tested: http://localhost:3000/admin/camp/62c098e7b4a7a433a7622ff4

Reply:
yeah we can remove this